api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
 {% extends "email_body_base.html" %}
 
-{% block swedish %}
-<h1>Hej {{member.firstname}},</h1>
-<p style="font-size: small;"><em>(For information in ENGLISH, see below the dashed line).</em></p>
-
-<p>
-    Välkommen till Stockholm Makerspace! Du är nu del av Sveriges största självständiga Makerspace. Tillsammans hjälps vi åt att förbli en självständig och medlems-driven teknisk fristad för gemenskap och skapande.</p>
-
-<p>Allt i Stockholm Makerspace är byggt, köpt, dit-kört och renoverat av medlemmar som du och jag. Ingen får lön och vi får inga bidrag. Orsaken till att vi kan betala hyran är att vi är många som hjälps åt genom att betala medlemsskap. Det är fantastiskt vad medborgare kan skapa tillsammans. Om du tycker att det finns något som kan utvecklas så är du varmt välkommen att engagera dig i driften av vår förening.
-</p>
-
-<p>
-    Vi vet av erfarenhet att du, som ny medlem, har mängder av frågor om Stockholm Makerspace. I detta mail hittar du den information du behöver för att själv kunna hitta svaren på alla dina frågor och funderingar, så läs mailet noga!
-</p>
-
-<h3>Medlemsinformation</h3>
-<p>Namn: <i>{{member.firstname}} {{member.lastname}}</i><br/>
-    Email: <i>{{member.email}}</i><br/>
-    Medlemsnummer: <i>{{member.member_number}}</i><br/>
-</p>
-
-<p>
-    Om föreningen behöver nå dig så använder vi den här emailen. Det är därför viktigt att du tar ansvar för att den här e-mailadressen fungerar, eller anmäler ny email. Medlemsnummret används för vår interna organisation och du kommer att behöva det för att anmäla dig till våra kurser.
-</p>
+<meta charset="UTF-8">
+<style>
+    .text-box {
+        border: 1px solid #000;
+        padding: 5px;
+    }
+  </style>
+
+{% block language clarification %}
```

Comment:
Is this block, `language clarification`, used for anything? As far as I understand, the other blocks are inserted into the template `{% extends "email_body_base.html" %}`. But this block is not defined in the template, so it will never be displayed, right?